test/python/circuit/test_circuit_load_from_qasm.py

```diff
@@ -108,9 +130,28 @@ def test_loading_all_qelib1_gates(self):
         ref_circuit.crz(0.6, 0, 1)
         ref_circuit.rxx(0.2, 0, 1)
         ref_circuit.rzz(0.2, 0, 1)
-        ref_circuit.measure([0, 1, 2], [0, 1, 2])
 
-        self.assertEqual(qasm_circuit, ref_circuit)
+        ref_circuit.rccx(0, 1, 2)
+        ref_circuit.rcccx(0, 1, 2, 3)
+        ref_circuit.append(C3XGate(), [0, 1, 2, 3])
+        ref_circuit.append(C3SXGate(), [0, 1, 2, 3])
+        ref_circuit.append(C4XGate(), [0, 1, 2, 3, 4])
+
+        ref_circuit.measure_all()
+
+        # check that ref_circuit contains all the operations from qasm_circuit
+        ref_ops = ref_circuit.count_ops().keys()
+        qasm_ops = qasm_circuit.count_ops().keys()
+        self.assertTrue(qasm_ops <= ref_ops, msg=qasm_ops - ref_ops)
```

Comment:
`self.assertLessEqual(qasm_ops, ref_ops)` should produce clearer logging output here (and below), but shouldn't they have exactly equal sets?